packages/expo-localization/android/src/main/java/expo/modules/localization/LocalizationUtils.kt

```diff
@@ -0,0 +1,40 @@
+package expo.modules.localization
+
+import android.text.TextUtils
+import java.lang.Exception
+import java.util.*
+import kotlin.collections.ArrayList
+
+val USES_IMPERIAL = listOf("US", "LR", "MM")
+
+val iSOCurrencyCodes: ArrayList<String> by lazy {
+  Currency.getAvailableCurrencies().map { it.currencyCode } as ArrayList<String>
```

Comment:
Same comment as for this one: https://github.com/expo/expo/pull/14588/files#r721602546. Why is casting `as ArrayList<String>` needed?